apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Snippets/SpannerConnectionSnippets.cs

```diff
@@ -424,8 +424,9 @@ public async Task WhenSessionPoolReady()
             // End sample
         }
 
-        [Fact]
-        public void GetSessionPoolDatabaseStatistics()
+        [Fact]
+        [Obsolete]

```

Comment:
Same comment as before re: not marking a test as Obsolete but using "ignore warning" instead.

But also, we should test the new method as well.